src/Features/CSharp/Portable/ChangeSignature/UnifiedArgumentSyntax.cs

```diff
@@ -47,7 +47,7 @@ public IUnifiedArgumentSyntax WithNameColon(SyntaxNode nameColonSyntax)
         }
 
         public string GetName()
-            => NameColon == null ? string.Empty : ((NameColonSyntax)NameColon).Name.Identifier.ToString();
+            => NameColon == null ? string.Empty : ((NameColonSyntax)NameColon).Name.Identifier.ValueText;
```

Comment:
do we have the same issue for VB?  can you add a test?  note: in vb, the form is `[new]`.

Reply:
issue doesn't repro in VB -- added a test just to be safe